irispy/new_sji.py

```diff
@@ -272,12 +272,9 @@ def __repr__(self):
            seq_shape=self.dimensions,
            axis_types=self.world_axis_physical_types)
 
-
     def __getitem__(self, item):
         return utils.sequence.slice_sequence(self, item)
```

Comment:
I use `utils.sequence.slice_sequence(self, item)` rather than `index_as_cube` because the last function is calling a lot of function that are not relevant (creating as many error as functions) and return the same result. What do you think about it ?